assets/javascripts/src/modules/analytics/ga.es6

```diff
@@ -16,8 +16,10 @@ const dimensions = {
     intcmp: 'dimension12'
 };
 
+const defaultTracker = 'membershipPropertyTracker';
+var gaEnabled = true;
```

Comment:
I'd use the Redux store. At initialisation you can dispatch an action to either enable or disable tracking, and then use `store.getState()` to read that state whenever you need it.